src/main/sensors/acceleration.c

```diff
@@ -42,6 +42,7 @@
 acc_t acc;                       // acc access functions
 int32_t accADC[XYZ_AXIS_COUNT];
 sensor_align_e accAlign = 0;
+const uint32_t accTargetLooptime = 1000;
```

Comment:
we can't assume acc is sampled at 1khz rate. it's sampled at the same rate as gyro at the moment.